oclint-json-compilation-database

```diff
@@ -52,8 +52,9 @@ if not source_exist_at(OCLINT_BIN):
     print("Error: OCLint executable file not found.")
     sys.exit(99)
 
+JSON_COMPILATION_DATABASE = os.path.join(args.build_path, 'compile_commands.json')
```

Comment:
Not quite sure the naming convention in Python, but since this is no longer a global variable, should we use lower cases?